### PR TITLE
Improve flag of Dominica

### PR DESCRIPTION
**Pull request by pyfisch**
* vertical and horizontal stripes are centered
* white stripe has the same width as the others
* no green ring on emblem
* use only one kind of green
* small 10 stars are properly located
* decrease size of emblem (still bigger than the reference)
* rotate bird 10° clockwise to so it fits with the stars

### CODE REVIEW

**Comment by b-g** [comment]
Hi @pyfisch,

many thanks for this! 

Did you test the flag with the OpenMoji Tester App https://openmoji-tester.glitch.me/? (Please read also https://github.com/hfg-gmuend/openmoji/blob/master/CONTRIBUTING.md)

cc @dnlutz OK with this level of detail?

**Comment by pyfisch** [comment]
Yes I've run the tester app and read the contributing file before submitting the PR.

**Comment by b-g** [comment]
Hi @pyfisch, many thanks for this! However we can't merge your contribution as the basic shapes are styled with an odd mix of attributes in combination with inline styles e.g.:

2019-10-15 at 10 54

This would break our entire workflow of testing and generating things. I've updated the openmoji-tester app to reflect this. The file is tested red now.

May I ask how you edited/exported the svg file?

Basically:
```html
bad: <rect style="fill:#5c9e31;" height="10" width="10" y="0" x="0"/> 
good: <rect fill="#5c9e31" height="10" width="10" y="0" x="0"/>
```

**Comment by pyfisch** [comment]
Hi, I've edited the file with Inkscape and exported it as standard SVG. Did not know you don't want inline styles but I've converted them with [SVGOMG](https://jakearchibald.github.io/svgomg/) to attributes.

**Comment by b-g** [comment]
Hi @pyfisch, brilliant! Merged! Will add your credits to the changelog.txt. Many thanks for the contribution!
